api-samples/tabs/screenshot/screenshot.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tab Screenshot</title>
+  </head>
+
+  <body>
+    Image here:
+    <p>
+      <img
+        id="target"
+        src="white.png"
+        height="480"
+        style="border: black solid 1px"
+      />
+    </p>
+
+    <p>
+      <script src="screenshot.js"></script>
```

Comment:
Does this script need to be in a paragraph tag?

Reply:
Done. https://github.com/GoogleChrome/chrome-extensions-samples/pull/964/commits/6c99f4b311ea4bc055bc4227964c68031da6516a